airflow/auth/managers/simple/ui/src/test-utils.tsx

```diff
@@ -18,23 +18,29 @@
  */
 
 import React, { PropsWithChildren } from "react";
-import { ChakraProvider } from "@chakra-ui/react";
+import { ChakraProvider, defaultSystem } from "@chakra-ui/react";
```

Comment:
I think ideally we would want to re-use the one from the main app, but I don't think we solved that problem yet did we ? (shareable front-end code from core to provider ?)

Reply:
I wish too :/ But yeah I could not find a solution